src/app/manage/distribution-centers.ts

```diff
@@ -21,9 +21,8 @@ export class DistributionCenters extends IdEntity {
   phone1Description = new StringColumn(getLang(this.context).phone1Description);
   phone2 = new PhoneColumn(getLang(this.context).phone2);
   phone2Description = new StringColumn(getLang(this.context).phone2Description);
-
-
-
+  isNotActive = new BoolColumn(getLang(this.context).frozen);
```

Comment:
added as negative logic assuming the default for bool is false and will not affect existing systems.

Reply:
Why not stay consistent with the archive concept that is used for helpers and FamilyDeliveries?